tests/test_cases/test_cocotb/common.py

```diff
@@ -45,10 +45,13 @@ def _check_traceback(running_coro, exc_type, pattern):
 
 
 @contextmanager
-def assert_raises(exc_type):
+def assert_raises(exc_type, pattern=None):
```

Comment:
I suppose at some point we may move (some of) this module out of test_cocotb to be more widely accessible, and then a docstring would be nice.